orchestration/OrchestrationService/CustomScriptExecution.ps1

```diff
@@ -0,0 +1,257 @@
+Class CustomScriptExecution {
+
+    hidden $scriptType = "";
+    hidden $command = "";
+    hidden $arguments = @{};
+    
+    # Used to store results from the script execution
+    $Result = "";
+    
+    CustomScriptExecution([string] $command, [hashtable] $arguments) {
```

Comment:
Move these arguments away from the constructor. The reason is because in the instance that we want to run multiple custom scripts, with this approach you'll end up creating multiple new instances of this class. As a rule of thumb, pass as constructor parameters whatever the class requires to exist/function, in this case, commands and arguments are variables and not really necessary for this class to exist, instead, you can pass these values to Execute() function

Reply:
Remove the arguments from constructor to execute method.